### ISSUE TITLE
Descriptions in English are not marked when Czech is selected

### ISSUE THREAD

**Issue by kosarko** [ufal/dspace-angular#16]
[3.1.2 Language of Parts](https://www.w3.org/TR/WCAG21/#language-of-parts)

Descriptions in English are not marked when Czech is selected

we should be able to mark descriptions as english either by using metadata or defaulting to english? That's a reasonable default for lindat, but for others?
